src/data_traits.rs

```diff
@@ -11,7 +11,12 @@
 #[allow(unused_imports)]
 use rawpointer::PointerExt;
 
+#[cfg(not(feature = "portable-atomic"))]
 use alloc::sync::Arc;
+
+#[cfg(feature = "portable-atomic")]
+use portable_atomic_util::Arc;
```

Comment:
Does this affect user experience at all, does it change the API?

It's encapsulated right, so almost nothing can change - fortunately - but if I didn't check this then something like Send- or Sync-ness of some types could change or something like that.

Reply:
I don't think Send or Sync should be modified, as far as I can tell here is the only possible public APIs that might be impacted

1. ArcArray
2. OwnedArcRepr (which according to the code comments should never be used by the user)

```Rust
/// ArcArray's representation.
///
/// *Don’t use this type directly—use the type alias
/// [`ArcArray`] for the array type!*
#[derive(Debug)]
pub struct OwnedArcRepr<A>(Arc<OwnedRepr<A>>);
```

`ArcArray` as far as I can see, never/rarely gives unfettered access to the `OwnedRepr` which has the `Arc`, as such I really don't think there is any affect to user experience, other than idiosyncratic different behavior from portable-atomic-util, which is unlikely, but possible.